modules/webadmin.cpp

```diff
@@ -1567,6 +1569,13 @@ class CWebAdminMod : public CModule {
                 o12["Checked"] = "true";
             }
 
+            CTemplate& o13 = Tmpl.AddRow("OptionLoop");
+            o13["Name"] = "builtinauthdisabled";
+            o13["DisplayName"] = t_s("Built-in Authentication Disabled");
```

Comment:
What does Built-in mean? I know what you mean by this, but someone who's looking at the UI can e.g. mistake this by "enable anonymous login without any password"... and what @psychon said makes this mistake even more dangerous.

Reply:
That's a good point, I don't have many better ideas on this. Would making this admin-only and documenting it on the wiki be an acceptable fix?

Reply:
Renaming to "OnlyModulesAuth" and having help text like "Only allow authentication with a module" would fix this problem as well.